### DISCUSSION TITLE
Fluidd printer.cfg 't5uid1’ error

### DISCUSSION THREAD

**Discussion by bmccullar** [Desuuuu/klipper#56] [Q&A] [answered]
I'm attempting to use Fluidd with your configuration (with touchscreen support) but upon launching the Fluidd UI I'm encountering the error "Section 't5uid1' is not a valid config section". I am not able to proceed forward without removing the config section which will disable touchscreen functionality.

I'm using the CR10SProV2 config.

**Comment by Desuuuu** [comment, marked as answer]
This error means you're not running my fork on your host.

**Reply by bmccullar** [reply]
You are correct, I forgot to use your fork instead of the main fork.
